core/scss/elements/link/_link.scss

```diff
@@ -5,21 +5,10 @@
 //
 // Default link styling, as well as special link treatment styling.
 //
-// .su-external-link  - External Link
-// .su-more-link      - More Link
-//
-// Markup: ../../templates/elements/link/link.twig
+// Markup: ../templates/elements/link/link.twig
 //
 // Style guide: Elements.Link
 //
 a {
   @include link;
 }
-
-.su-external-link {
```

Comment:
I removed these in order to move them up into the link component with the other variants.